webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/campaigns/DataTypes.scala

```diff
@@ -84,14 +84,43 @@ object CampaignId                                                            {
   }
 }
 
+sealed trait CampaignStatusValue {
+  def value: String
+}
 @jsonDiscriminator("value")
-sealed trait CampaignStatus
+sealed trait CampaignStatus      {
+  def value: CampaignStatusValue
+}
+
+object CampaignStatus {
```

Comment:
Can you call the companion object like the trait (ie `CampaignStatusValue`) else it's really confusing and against convention. Plus I'm pretty sure it will cause problems with the actual `CampaignStatus` trait & objects